units/deco_host_notify.go

```diff
@@ -134,11 +132,10 @@ func (j *decoHostNotifyJob) Run(_ context.Context) {
 	issue := message.JiraIssue{
 		Project:     conf.Jira.DefaultProject,
 		Summary:     fmt.Sprintf("investigate automatically decommissioned host '%s'", j.Host.Id),
```

Comment:
@zakhark do these go to your team? Do you still want these to be ticketed? I think this would be better as a splunk alert.

Reply:
(I think this about the s3 creds as well but that one i could see needing to go to jira more)